recipes/sphinx_bootstrap_theme/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sphinx-bootstrap-theme" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "630401f27ece7a49ae3d8f028cf9e885" %}
```

Comment:
This is a `mdf5` and not a `sha256`. That is the trap of all that jinja vars everywhere...